ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,32 @@
-﻿namespace ConsoleApp1 {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿using System;
+
+namespace PalindromeChecker {
+   class Program {
+      static bool IsPalindrome (string input) {
+         input = input.ToLower ().Replace (" ", "").Replace (".", "").Replace (",", "").Replace ("?", "").Replace ("'", "").Replace ("!", "");
+         int left = 0;
+         int right = input.Length - 1;
+
+         while (left < right) {
+            if (input[left] != input[right]) {
+               return false;
+            }
+            left++;
+            right--;
+         }
+
+         return true;
+      }
+
+      static void Main (string[] args) {
+      Console.WriteLine ("Enter a sentence or word:");
+      string input = Console.ReadLine ();
+
+         if (IsPalindrome (input)) {

```

Comment:
Use correct alignment. Here the 'if' block is 'tabbed' right. 
One thing you can do is type "CTRL + k + d" shortcut to apply the coding style automatically. Ping me if you have doubts regarding this.

Reply:
thanks for notifying the corrections .I will make the corrections and push again after making corrections